staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -149,6 +149,19 @@ func dataHandler(ctx context.Context, dev *driver.CustomizedDev) {
 			ReportToCloud:   twin.Property.ReportToCloud,
 		}
 		go twinData.Run(ctx)
+
+		//handle status
+		getStates := DeviceStates{Client: dev.CustomizedClient, DeviceName: dev.Instance.Name,
+			DeviceNamespace: dev.Instance.Namespace}
+		// TODO setting status reportCycle
+		timer := common.Timer{Function: getStates.Run, Duration: 2 * time.Second, Times: 0}
+		var wg sync.WaitGroup
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			timer.Start()
+		}()
```

Comment:
Is it possible to imitate the code reported in devicetwin? that is, use a goroutine `go getstates.run()`, and then start a timer in a similar way in the devicetwin file like:
![twinreport](https://github.com/kubeedge/kubeedge/assets/48212997/38c123f7-c794-4068-aa99-0c6aa21fb6d5)